core/config/blockchain_info.go

```diff
@@ -12,7 +12,12 @@ type BlockchainInfo struct {
 	Name    string
 	// IsEtherBased is true for chains that use Ether as the currency for fees.
 	IsEtherBased bool
-	Blocktime    time.Duration
+	// IsEthereum applies to ethereum mainnet and all testnets. This setting is
+	// mainly used for testing features on our test networks, as towns mainnet
+	// should not have any test networks configured and only mainnet ethereum
+	// should qualify.
+	IsEthereum bool
```

Comment:
I want to suggest that you call this IsMainnet instead of IsEthereum, so you can just stick with ethChainIds and mainnetChainIds, but it's also testnet... so idk. I guess this is fine.